pywren/wrenhandler.py

```diff
@@ -294,11 +294,21 @@ def generic_handler(event, context_dict):
         CONDA_PYTHON_PATH = "/tmp/condaruntime/bin"
         CONDA_PYTHON_RUNTIME = os.path.join(CONDA_PYTHON_PATH, "python")
 
-        cmdstr = "{} {} {} {} {}".format(CONDA_PYTHON_RUNTIME,
-                                         jobrunner_path,
-                                         func_filename,
-                                         data_filename,
-                                         output_filename)
+        # pass a full json blob
+        jobrunner_config_filename = "/tmp/jobrunner.config.json"
```

Comment:
can we make this a constant at the top of the file ? Also generating a unique name might be good ?